roles/opensrp/tasks/vietnam.yml

```diff
@@ -0,0 +1,90 @@
+---
+# Required variables:
+#  brand_login_page
+- name: Delete login header file
+  become: yes
+  become_user: "root"
+  file:
+    state: absent
+    path: "{{ opensrp_user_home }}/{{ opensrp_tomcat_instance }}/webapps/opensrp/WEB-INF/jspf/header.jspf"
+
+- name: Copy custom header.jspf to tomcat
```

Comment:
Could we make this task optional since I'm assuming not all deployments need it. IMO, it would also be nice if the template file were in your inventory and not the role. You can achieve that by making whatever is the source in this task a variable, then setting the path to the file in your inventory (to a file in your inventory).

Same applies for all the other files being copied from the `tmp/miecd_branding` directory.

Reply:
The task will only be executed if `brand_login_page` param is set in inventory. Let me work on moving the branding files to inventory.

Reply:
@jasonrogena how is it now? I have moved `tmp/miecd_branding` to inventory